docs/pages/setup/helm-reference/teleport-cluster.mdx

```diff
@@ -63,51 +63,82 @@ This reference details available values for the `teleport-cluster` chart.
 
 `kubeClusterName` sets the name used for the Kubernetes cluster. This name will be shown to Teleport users connecting to the cluster.
 
-## `authenticationType`
+## `authentication`
+
+### `authentication.type`
 
 | Type | Default value | Required? | `teleport.yaml` equivalent | Can be used in `custom` mode? |
 | - | - | - | - | - |
 | `string` | `local` | Yes | `auth_service.authentication.type` | ❌ |
 
-`authenticationType` controls the authentication scheme used by Teleport. Possible values are `local` and `github` for OSS, plus `oidc`, `saml`, and `false` for Enterprise.
+`authentication.type` controls the authentication scheme used by Teleport. Possible values are `local` and `github` for OSS, plus `oidc` and `saml` for Enterprise.
+
+### `authentication.connectorName`
+
+| Type | Default value | Required? | `teleport.yaml` equivalent | Can be used in `custom` mode? |
+| - | - | - | - | - |
+| `string` | `""` | No | `auth_service.authentication.connector_name` | ❌ |
+
+`authentication.connectorName` sets the default authentication connector.
+See [the SSO documentation](../../enterprise/sso.mdx). Other possible values are `local` and
+[`passwordless`](../../access-controls/guides/passwordless.mdx#optional-enable-passwordless-by-default).
+Defaults to `local`.
+
+### `authentication.localAuth`
+
+| Type | Default value | Required? | `teleport.yaml` equivalent | Can be used in `custom` mode? |
+| - | - | - | - | - |
+| `bool` | `true` | No | `auth_service.authentication.local_auth` | ❌ |
+
+`authentication.localAuth` controls whether local authentication is enabled.
+When disabled, users can only log in through authentication connectors like `saml`, `oidc` or `github`.
+
+[Disabling local auth is required for FedRAMP / FIPS](../../enterprise/fedramp.mdx#teleport-auth-server).
+
+### `authentication.lockingMode`
+
+| Type | Default value | Required? | `teleport.yaml` equivalent | Can be used in `custom` mode? |
+| - | - | - | - | - |
+| `string` | `""` | No | `auth_service.authentication.locking_mode` | ❌ |
 
-## `authenticationSecondFactor`
+`authentication.lockingMode` controls the locking mode cluster-wide.
+See [the locking modes documentation](../../access-controls/guides/locking.mdx#next-steps-locking-modes)
```

Comment:
Since there are only two possible values, I think we can list/describe these here, rather than referring the user to another docs page.

Reply:
Done